Doc/library/typing.rst

```diff
@@ -2059,6 +2057,10 @@ Introspection helpers
    .. versionchanged:: 3.9
       Added ``include_extras`` parameter as part of :pep:`593`.
 
+   .. versionchanged:: 3.11
+      ``Optional`` annotation is not added implicitly anymore
+      when ``None`` default is used for a function argument.
```

Comment:
I like the wording in the removed docs better. Suggestion:

"Previously, ``Optional[t]`` was added for function and method annotations if a default value equal to ``None`` was set. Now the annotation is returned unchanged."

Reply:
Great addition!